File: Simulator-CSharp/Models/ATM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace ATM.Models
{
    public class ATM
    {
                
        public TimeSpan Alive { get; set; }
        public TimeSpan StandBy { get; set; }

        public String Bank { get; set; }
        public String Code { get; set; }

        public List<Box> Boxes { get; set; }

        public int Receipts { get; set; }

        public override string ToString()
        {
            return Bank + ": " + Code;
        }

    }
}

File: Simulator-VB/Models/ATM.vb

Namespace Models

    Public Class ATM

        Public Property Active As TimeSpan

        Public Property Code As String

        Public Property Bank As String

        Public Property Boxes As List(Of Box)

        Public Property Receipts As Integer

        Public Overrides Function ToString() As String
            Return Me.Code & "; " & Me.Bank
        End Function

    End Class

End Namespace